style.css

```diff
@@ -1 +1,121 @@
 /*  Add your styling here */
+
+*{
+    margin:0;
+    padding:0;
+}
+ body{
+    display: grid;
+    grid-template-areas: "header" "main" "footer";  
+    font-family: Arial, Helvetica, sans-serif;
+}
+
+
+header{
+    padding: 1rem;
+    grid-area: header;
+}
+
+.header{
+    display: grid;
+    grid-template-areas: "logo hamburger" "logo intro";
+}
+
+.logo{
+    grid-area: logo;
+}
+
+.hamburger{
+    grid-area: hamburger;
+    border: none;
+    background: transparent;
+    text-align: end;
+}
+
+.intro{
+    grid-area: intro;
+    text-align: end;
+}
+
+nav{
+    display: none;
+}
+main{   
+display: grid;
+grid-area: main;
+grid-template-columns: 1fr 1fr;
+grid-template-areas: "welcome welcome" "info info" "home-img home-img" "services services" "cheese birthday" "cupcakes specials";
```

Comment:
I would push this logic down to make it easier to understand.  As in make it the responsibility of the individual containers to determine how their items are rendered.  

So for example, create a container for around your "cheese cake / birthday cake / etc items" and make that have the grid layout you desire. (Two items wide, equally shared)